### ISSUE TITLE
Keyboard support for MultiSelect component

### ISSUE THREAD

**Issue by bjovanov** [primefaces/primeng#6025]
[x] feature request

Currently the MultiSelect component does not have keyboard support at all. Due to the convenience of filling out forms using keyboard only, it would be great if the multiselect can be manipulated with keyboard. I have tried the autocomplete component for this purpose, but the multiselect fits better. Possible way of implementing this would be to open the multiselect on keypress and enter the typed letter in the filter, use up/down keys to navigate through items as it is done in the dropdown component and use space to check/uncheck items.

**Comment by cagataycivici** [comment]
Planned for 7.0.

**Comment by shumail92** [comment]
@cagataycivici Any idea on timeline of 7.0?
